common/src/mbus_api/mod.rs

```diff
@@ -381,6 +381,18 @@ impl From<JoinError> for ReplyError {
     }
 }
 
+/// for errors that can occur when serializing or deserializing JSON data
+impl From<serde_json::Error> for ReplyError {
```

Comment:
We've hardcoded Unknown which works for JsonGrpc, but might not for other cases, where the error might pertain to something specific. Also, is Aborted the right error kind here?
Maybe a helper method would be more appropriate?